driver/Cargo.toml

```diff
@@ -12,7 +12,9 @@ path = "src/lib.rs"
 bson = "0.9.1"
 byteorder = "1.3.1"
 chrono = "0.4.6"
+clarity = "0.1.8"
 ethabi = "6.1.0"
+ethereum-tx-sign = { git = "https://github.com/synlestidae/ethereum-tx-sign.git" }
```

Comment:
Is there any particular reason we are pegged to a specific repo here? Does this imply the package is not published in cargo?

Reply:
Good point. We need the chain_id parameter in the sign functions (added in version 1.0.0 here: https://github.com/synlestidae/ethereum-tx-sign/pull/3/files). The latest version on Cargo is 0.0.2.

But I will pin to a specific commit/tag and created an issue: https://github.com/synlestidae/ethereum-tx-sign/issues/11